src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] ball = new Lottery[3];
```

Comment:
Lottery is lottery. Ball is ball. Lottery[] ball ? What does it mean? And why do you need array of Lotteries? Create array of Balls.